contentcuration/contentcuration/frontend/shared/vuex/session/index.js

```diff
@@ -85,14 +85,11 @@ export default {
     featureFlags(state) {
       return state.currentUser.feature_flags || {};
     },
-    hasFeatureEnabled(state, getters) {
-      /**
-       * @param {string} flag - shared.constants.FeatureFlagKeys.*
-       * @return {Boolean}
-       */
-      return function(flag) {
-        return getters.isAdmin || Boolean(getters.featureFlags[flag]);
-      };
+    isAIFeatureEnabled(state, getters) {
+      if (getters.loggedIn) {
+        return Boolean(getters.featureFlags['ai_feature']);
+      }
+      return false;
```

Comment:
This should be admin only. I also think that using the existing `hasFeatureEnabled` with a flag constant is preferred

Reply:
> This should be admin only. 

Means, the user should be an admin to access this getter? I'm not clear with what you mean sir.

> I also think that using the existing hasFeatureEnabled with a flag constant is preferred

Will refactor sir.

Reply:
@vkWeb I think what Blaine means here is that instead of getting rid of the `hasFeatureEnabled` getter, we could use it within our `isAIFeatureEnabled` getter ie.
```
isAIFeatureEnabled(state, getters) {
      if (getters.loggedIn) {
          return Boolean(getters.hasFeatureEnabled('ai_feature'))
      }
      return false;
}
```

Also, I think the `isAIFeatureEnabled` getter can only be accessed if you are a regular user with the AI feature turned on or if you are an admin. I am guessing this is something that was missed in the spec. But we can wait on Blaines clarification!

Reply:
Maybe I'm misremembering but I thought there was motivation to keep the 'AI Features' hardcoded to admins only (with the flag) for the initial roll out so we can ensure everything is working properly and we're not running up large costs. If we feel staging/hotfixes testing is sufficient for that, then we can remove the 'admin only' restriction.

Reply:
This has been cleared on Google meet, there was confusion regarding this.

**The getter can be accessed by any user & hasFeatureEnabled will be used** -- the way we will move forward.

Reply:
@vkWeb does this mean that we are proceeding with the PR's implementation or some changes need to be made before the merge can happen?

Reply:
Hmm I think there's still some confusion @vkWeb :) 

There are still changes needed here: `getters.isAdmin || ` needs to exist in any case. I also prefer we keep `hasFeatureEnabled` around so we can reuse it since we can have multiple feature flags at one time, and it leaves a pathway to default consolidation of the logic. For instance, not including `getters.isAdmin || ` which should always be applicable for any feature flag.

Reply:
@bjester sir the consfusion was cleared on our google meet and I already had prepared the code for it, just writing some tests. Maybe the way I put it didn't convey my thoughts clearly  -- 
> The getter can be accessed by any user & hasFeatureEnabled will be used

Anyways, commits landing soon!